internal/service/appfabric/app_authorization_test.go

```diff
@@ -0,0 +1,437 @@
+			acctest.PreCheck(ctx, t)
+		},
+		ErrorCheck:               acctest.ErrorCheck(t, names.AppFabricServiceID),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckAppAuthorizationDestroy(ctx),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAppAuthorizationConfig_tags1("key1", "value1updated"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAppAuthorizationExists(ctx, resourceName, &appauthorization),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"credential"},
+			},
+			{
+				Config: testAccAppAuthorizationConfig_tags2("key1", "value1updated", "key2", "value2"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAppAuthorizationExists(ctx, resourceName, &appauthorization),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"credential"},
+			},
+			{
+				Config: testAccAppAuthorizationConfig_tags1("key2", "value2"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAppAuthorizationExists(ctx, resourceName, &appauthorization),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"credential"},
+			},
+		},
+	})
+}
+
+func testAccCheckAppAuthorizationDestroy(ctx context.Context) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := acctest.Provider.Meta().(*conns.AWSClient).AppFabricClient(ctx)
+
+		for _, rs := range s.RootModule().Resources {
+			if rs.Type != "aws_appfabric_app_authorization" {
+				continue
+			}
+
+			_, err := tfappfabric.FindAppAuthorizationByTwoPartKey(ctx, conn, rs.Primary.Attributes[names.AttrARN], rs.Primary.Attributes["app_bundle_identifier"])
+
+			if tfresource.NotFound(err) {
+				continue
+			}
+
+			if err != nil {
+				return err
+			}
+
+			return fmt.Errorf("App Fabric App Authorization %s still exists", rs.Primary.ID)
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckAppAuthorizationExists(ctx context.Context, n string, v *types.AppAuthorization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		conn := acctest.Provider.Meta().(*conns.AWSClient).AppFabricClient(ctx)
+
+		output, err := tfappfabric.FindAppAuthorizationByTwoPartKey(ctx, conn, rs.Primary.Attributes[names.AttrARN], rs.Primary.Attributes["app_bundle_identifier"])
+
+		if err != nil {
+			return err
+		}
+
+		*v = *output
+
+		return nil
+	}
+}
+
+func testAccAppAuthorizationConfig_basic() string {
+	return `
+
+
+resource "aws_appfabric_app_authorization" "test" {
+  app_bundle_identifier = aws_appfabric_app_bundle.arn
```

Comment:
Where is the resource for this?

Reply:
This is still on creation the tests will be updated with the team that is working on that resource will make a pull request